src/inc/quic_platform_winuser.h

```diff
@@ -572,6 +572,67 @@ typedef HANDLE CXPLAT_EVENT;
 #define CxPlatEventWaitWithTimeout(Event, timeoutMs) \
     (WAIT_OBJECT_0 == WaitForSingleObject(Event, timeoutMs))
 
+//
+// Event Queue Interfaces
+//
+
+typedef HANDLE CXPLAT_EVENTQ;
+#define CXPLAT_SQE OVERLAPPED
```

Comment:
What do "SQE" and "CQE" stand for?

Reply:
"Submission queue event" and "completion queue event". Stolen from io_uring terminology.